upgrades/upgrade_testing_framework/steps/__init__.py

```diff
@@ -11,3 +11,6 @@
 from upgrade_testing_framework.steps.step_stop_target_cluster import StopTargetCluster
 from upgrade_testing_framework.steps.step_test_source_cluster import TestSourceCluster
 from upgrade_testing_framework.steps.step_test_target_cluster import TestTargetCluster
```

Comment:
Probably don't need these any more; how do you feel about removing them?

Reply:
Do you think you have any need for validating a cluster has started properly with these steps? I was originally thinking you might do some startup validation here or any checks that couldn't be accomplished through the API

Reply:
Good question!  Ideally, the test that stands up the clusters should not exit successfully unless they're properly started, yeah?  In that case, we still don't need these `Test*` steps.

Reply:
Hey, actually, changed my mind about these.  Let me remove these later, as I have some in-flight changes that will get messy if you remove them.